Prevent tooltip overflow (#507)
Blobscan/blobscan: .changeset/tricky-actors-worry.md

---
"@blobscan/web": patch
---

> The Tooltip component now includes extra logic to prevent it from overflowing.

Reviewer: Let's follow the standard and only use past tense verbs please

```suggestion
Added additional logic to the `Tooltip` component to prevent overflow.
```
